pkg/k8sutil/client.go

```diff
@@ -22,22 +22,6 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-// NewClientsetFromFile creates a new Kubernetes Client set object from the given
```

Comment:
is this commit [f74c7f7](https://github.com/kinvolk/lokomotive/pull/628/commits/f74c7f7c1d008d54c527cbe0f09b6c7514436ed2) before or after the current commit ? I guess after rebase github messes up the commit order.

If its after, then I'd change it to before to truly reflect the commit message
```
NewClientsetFromFile is now deprecated, as we ...
```

Reply:
First goes `cli/cmd: use NewClientset to construct Kubernetes clientset`, then `pkg/k8sutil: remove unused NewClientsetFromFile`. I'm not sure what change are you suggesting.

Reply:
Never mind, I meant Github was showing the wrong order to me